core/include/tee/fs_dirfile.h

```diff
@@ -56,7 +56,7 @@ struct tee_fs_dirfile_fileh {
  * @commit_writes:	commits changes since the file was opened
  */
 struct tee_fs_dirfile_operations {
-	TEE_Result (*open)(bool create, const TEE_UUID *uuid,
+	TEE_Result (*open)(bool create, uint8_t *hash, const TEE_UUID *uuid,
```

Comment:
Oh, I'm sorry. That was structure description, not function description. Please ignore this.